src/components/Validator.vue

```diff
@@ -38,9 +38,10 @@
                     <br />
                     <v-progress-circular :size="50" color="secondary" indeterminate v-show="processingFile"></v-progress-circular>
                     <v-alert :type="overallAlertType">{{ overallAlertMessage }}</v-alert>
-                    <v-alert v-show="hasFile && !processingFile" outlined :type="fileNameAlertType">{{ fileNameAlertMessage }}</v-alert>
-                    <v-alert v-show="hasFile && !processingFile" outlined :type="durationAlertType">{{ durationAlertMessage }}</v-alert>
-                    <v-alert v-show="hasFile && !processingFile" outlined :type="memoryAlertType">{{ memoryAlertMessage }}</v-alert>
+                    <v-alert v-show="hasFile && !processingFile && fileNameAlertMessage" outlined :type="fileNameAlertType">{{ fileNameAlertMessage }}</v-alert>
+                    <v-alert v-show="hasFile && !processingFile && durationAlertMessage" outlined :type="durationAlertType">{{ durationAlertMessage }}</v-alert>
+                    <v-alert v-show="hasFile && !processingFile && memoryAlertMessage" outlined :type="memoryAlertType">{{ memoryAlertMessage }}</v-alert>
+                    <v-alert v-show="hasFile && !processingFile && otherErrorAlertMessage" outlined :type="otherErrorAlertType">{{ otherErrorAlertMessage }}</v-alert>
```

Comment:
Not sure how to handle this in terms of UI. I'm thinking it might be better to combine all the same type of alerts to single alerts with a multi line message.

Reply:
Would be a possibility, although I think it would look better structured with the current approach.